src/main/app/claims/claimStoreClient.ts

```diff
@@ -235,7 +235,8 @@ export class ClaimStoreClient {
   }
 
   saveClaimantResponse (claim: Claim, draft: Draft<DraftClaimantResponse>, user: User): Promise<void> {
```

Comment:
@robertparkinson @varma02 @rattur pls review - these changes are to fix claimant response with refer to judge formalisation option (1 of 2)

Reply:
This looks good to me by passing isDefendantBusiness within the ClaimStoreClient and then checking in the ClaimantResponseConverter for isBusiness and setting REFER_TO_JUDGE